PANDORA/PMHC/Model.py

```diff
@@ -10,8 +10,8 @@
 
 class Model:
 
-    def __init__(self, target, model_path='', output_dir = PANDORA.PANDORA_data, pdb=False, molpdf=0, dope=0):
-        '''__init__(self, target, model_path='', output_dir = PANDORA.PANDORA_data, pdb=False, molpdf=0, dope=0)
+    def __init__(self, target, model_path='', output_dir = os.getcwd(), pdb=False, molpdf=0, dope=0):
+        '''__init__(self, target, model_path='', output_dir = os.getcwd(), pdb=False, molpdf=0, dope=0)
```

Comment:
Not that "PANDORA.PANDORA_data is not in the function definition anymore, you can remove this line. Its only role was to preven sphinx from printing the whole PANDORA.PANDORA_data path in the dcoumentation.